RecoVertex/AdaptiveVertexFit/src/AdaptiveVertexFitter.cc

```diff
@@ -632,9 +638,14 @@ AdaptiveVertexFitter::fit( const vector<RefCountedVertexTrack> & tracks,
          ( ((previousPosition - newPosition).mag() > theMaxShift) ||
            (!(theAssProbComputer->isAnnealed()) ) ) ) ;
 
-  if ( theWeightThreshold > 0. &&  ns_trks < 2 && !withPrior ) 
+  if ( theWeightThreshold > 0. &&  ns_trks < 1 && !withPrior ) 
```

Comment:
the cut was there since 2006. Why change it now?
maybe withPrior should be set?